tests/integration/llm/client.py

```diff
@@ -229,6 +237,26 @@ def test_ds_raw_model(model):
                 assert float(memory) / 1024.0 < spec["max_memory_per_gpu"]
 
 
+def test_ds_aot_handler(model):
```

Comment:
what is special for AOT Handler? Could we just use DS raw handler?

Reply:
Re-using DS raw handler now